src/docs/save.md

```diff
@@ -1,11 +1,18 @@
 Write the documentation stored in `modulename` to the specified `file`.
-The format is guessed from the file's extension. Currently supported formats
-are `HTML` and `markdown`.
+The format is guessed from the file's extension. Currently supported formats are `HTML` and
+`markdown`.
+
+**Returns**
+
+* `::Config` : the configuration which can be used as input argument to the related `savegenindex`
+  function.
 
 #### Options
 
 *General Options*
 
+* `category_order` (default: `:module, :function, :method, :type, :typealias, :macro, :global, :comment`)
```

Comment:
Wrapping the default value in brackets, `[:module, ..., :comment]`, to show that it is a vector would be good here.